addons/website/controllers/form.py

```diff
@@ -279,6 +278,25 @@ def insert_record(self, request, model, values, custom, meta=None):
                     message_type='comment',
                 )
 
+        if hasattr(record, '_message_log'):
+            warning_icon = ""
+            authenticate_message = False
+            if request.session.uid:
+                model_record = request.env[model_name]
+                user_email = request.env.user.email
+                form_email = values[model_record._mail_get_primary_email_field()]
+                if user_email != form_email:
+                    authenticate_message = _("This %(model_name)s was submitted by %(user_name)s (%(user_email)s) on behalf of %(form_email)s",
+                        model_name=model.name, user_name=request.env.user.name, user_email=user_email, form_email=form_email)
+            else:
+                warning_icon = "/!\\ "
+                authenticate_message = _("EXTERNAL SUBMISSION - Customer not verified")
+            if authenticate_message:
+                record._message_log(
+                    body=Markup('<div class="alert alert-info" role="alert">{warning_icon}{message}</div>').format(warning_icon=warning_icon, message=authenticate_message),
+                    author_id=request.env['ir.model.data']._xmlid_to_res_id('base.partner_root'),
```

Comment:
@xmo-odoo 
Since we just need the id, I thought it would be enough to use that method. By doing that, we avoid browsing and calling [`exists` method](https://github.com/odoo/odoo/blob/8136f61949ed6d4b72a7f3834633340b433043c0/odoo/api.py#L589). 🤷🏼‍♂️ 

But I am not against to use `ref` if you think it is better. 🙂